src/number_theory/pell.lean

```diff
@@ -41,103 +41,6 @@ Pell's equation
 
 namespace pell
 
-section existence
```

Comment:
For the sake of review, could you make a quick PR where you move this section with no changes?

Reply:
A separate PR would be best I think

Reply:
You can just cherry-pick that commit to create it.

Reply:
How do I do that (I mean create a PR from this commit)?

Reply:
Here's an outline:
* Create a new branch starting at master
* Run `git cherry-pick 6669f49f41b015344538f3200be44592443174df`
* Push the branch, create the PR

Reply:
OK, done: #18639.

Reply:
Should I now merge master?

Reply:
Yes, please do!